src/app/src/main/java/com/cmput301f21t09/budgetprojectname/services/database/DatabaseService.java

```diff
@@ -0,0 +1,16 @@
+package com.cmput301f21t09.budgetprojectname.services.database;
+
+import com.google.firebase.database.FirebaseDatabase;
+
+public class DatabaseService {
+
+    private static final FirebaseDatabase db = FirebaseDatabase.getInstance();
```

Comment:
I think we were switching over to Firestore so should be FireBaseFireStore db = ...

Reply:
Whoops, I thought I switched branches

Reply:
Fixed